core/modules/field/modules/options/css/options.css

```diff
@@ -0,0 +1,102 @@
+
+div.options-widget .draggable a.tabledrag-handle {
+  padding-right: .3em /* RTL */
+}
+
+div.form-options div.options {
+  margin-bottom: 0;
+}
+
+div.options-widget a.add span,
+div.options-widget a.remove span {
+  display: none;
+}
+
+div.options-widget a.add,
+div.options-widget a.remove,
+div.form-option-add a {
+  display: block;
+  width: 16px;
+  height: 16px;
+  margin: .2em .1em;
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  float: left; /* RTL */
+}
+
+div.options-widget a.add:hover,
+div.options-widget a.remove:hover,
+div.form-option-add a:hover {
+  background-position: 0 -16px;
+}
+
+div.options-widget a.add {
+  background-image: url('../images/add.png');
+}
+
+div.options-widget a.remove {
+  background-image: url('../images/delete.png');
+}
+
+div.default-value-pattern-match {
+  float: left; /* RTL */
+}
+
+div.remove-default {
+  float: left; /* RTL */
+}
+
+div.form-options-manual,
+div.form-option-add {
+  text-align: right; /* RTL */
+}
+div.form-options-manual a,
+div.form-option-add a {
+  float: none;
+  display: inline;
+  padding-left: 20px; /* RTL */
```

Comment:
This padding-left should be removed for the `.form-options-manual` selector. It results in a weird extra padding when the Manual entry link is focused:

![image](https://user-images.githubusercontent.com/100206/44961771-5d32bb00-aecb-11e8-8eac-3078b39a0273.png)

Reply:
removed